util/inteltool/gpio_names/geminilake.h

```diff
@@ -0,0 +1,311 @@
+	"*GPIO_165",	"AVS_I2S1_SDO",		"N/A",				"CNV_BT_I2S_SD0",
+	"*GPIO_166",	"AVS_HDA_B_CLK",	"AVS_I2S2_BCLK",	"N/A",
+	"*GPIO_167",	"AVS_HDA_WS_SYNC",	"AVS_I2S2_WS_SYNC",	"N/A",
+	"*GPIO_168",	"AVS_HDS_SDI",		"AVS_I2S2_SDI",		"N/A",
+	"*GPIO_169",	"AVS_HDA_SDO",		"AVS_I2S2_SDO",		"N/A",
+	"*GPIO_170",	"AVS_HDA_RST_N",	"AVS_I2S1_MCLK",	"N/A",
+	"*GPIO_171",	"AVS_DMIC_CLK_A1",	"N/A",				"N/A",
+	"*GPIO_172",	"AVS_DMIC_CLK_B1",	"N/A",				"N/A",
+	"*GPIO_173",	"AVS_DMIC_DATA_1",	"N/A",				"N/A",
+	"*GPIO_174",	"AVS_DMIC_CLK_AB2",	"N/A",				"N/A",
+	"*GPIO_175",	"AVS_DMIC_DATA_2",	"N/A",				"N/A",
+};
+
+static const char *const glk_group_scc_names[] {
+	"*GPIO_176",	"SMB_ALERT_N",		"N/A",				"N/A",
+	"*GPIO_177",	"SMB_CLK",			"SIO_I2C7_SCL",		"N/A",
+	"*GPIO_178",	"SMB_DATA",			"SIO_I2C7_SDA",		"N/A",
+	"*GPIO_179",	"SDACARD_CLK",		"N/A",				"N/A",
+	"*GPIO_181",	"SDCARD_D0",		"N/A",				"N/A",
+	"*GPIO_182",	"SDCARD_D1",		"N/A",				"N/A",
+	"*GPIO_183",	"SDCARD_D2",		"N/A",				"N/A",
+	"*GPIO_184",	"SDCARD_D3",		"N/A",				"N/A",
+	"*GPIO_185",	"SDCARD_CMD",		"N/A",				"N/A",
+	"*GPIO_186",	"SDCARD_CD_N",		"N/A",				"N/A",
+	"*GPIO_187",	"SDCARD_LVL_WP",	"N/A",				"N/A",
+	"*GPIO_188",	"SDCARD_PWR_DWN_N",	"N/A",				"N/A",
+	"*GPIO_210",	"N/A",				"N/A",				"N/A",
+	"*GPIO_189",	"OSC_CLK_OUT_0",	"N/A",				"N/A",
+	"*GPIO_190",	"OSC_CLK_OUT_1",	"N/A",				"N/A",
+	"*GPIO_191",	"CNV_BRI_DT",		"N/A",				"SIO_UART1_RTS_N",
+	"*GPIO_192",	"CNV_BRI_RSP",		"N/A",				"SIO_UART1_RXD",
+	"*GPIO_193",	"CNV_RGI_DT",		"N/A",				"SIO_UART1_TXD",
+	"*GPIO_194",	"CNV_RGI_RSP",		"N/A",				"SIO_UART1_CTS_N",
+	"*GPIO_195",	"CNV_RF_RESET_N",	"N/A",				"AVS_I2S1_WS_SYNC",
+	"*GPIO_196",	"XTAL_CLKREQ",		"N/A",				"AVS_I2S1_SDO",
+	"*GPIO_198",	"EMMC_CLK",			"N/A",				"N/A",
+	"*GPIO_200",	"EMMC_D0",			"N/A",				"N/A",
+	"*GPIO_201",	"EMMC_D1",			"N/A",				"N/A",
+	"*GPIO_202",	"EMMC_D2",			"N/A",				"N/A",
+	"*GPIO_203",	"EMMC_D3",			"N/A",				"N/A",
+	"*GPIO_204",	"EMMC_D4",			"N/A",				"N/A",
+	"*GPIO_205",	"EMMC_D5",			"N/A",				"N/A",
+	"*GPIO_206",	"EMMC_D6",			"N/A",				"N/A",
+	"*GPIO_207",	"EMMC_D7",			"N/A",				"N/A",
+	"*GPIO_208",	"EMMC_CMD",			"N/A",				"N/A",
+	"*GPIO_209",	"EMMC_RCLK",		"N/A",				"N/A",
+};
+
+static const struct gpio_group glk_group_north = {
+	.display	= "------- GPIO Group North -------",
+	.pad_count	= ARRAY_SIZE(glk_group_north_names) / 6,
+	.func_count	= 6,
+	.pad_names	= glk_group_north_names,
+};
+
+static const struct gpio_group *const glk_community_north_groups[] = {
+	&glk_group_north,
+};
+
+static const struct gpio_community glk_community_north = {
+	.name		= "----- GPIO Community North -----",
+	.pcr_port_id	= 0xc5,
+	.group_count	= ARRAY_SIZE(glk_community_north_groups),
+	.groups		= glk_community_north_groups,
+};
+
+static const struct gpio_group glk_group_northwest = {
+	.display	= "----- GPIO Group NorthWest -----",
+	.pad_count	= ARRAY_SIZE(glk_group_northwest_names) / 4,
+	.func_count	= 4,
```

Comment:
```suggestion
	.func_count	= 6,
```

There are 6 alternative functions in total

Reply:
Shouldn't it be 7? NorthWest group can have 7 alternative functions including the default.

Reply:
Right, my bad.